sensors.py

```diff
@@ -72,8 +125,9 @@ class subfeature(Structure):
 def init(cfg_file = None):
     file = _libc.fopen(cfg_file.encode("utf-8"), "r") if cfg_file is not None else None 
     
-    if _hdl.sensors_init(file) != 0:
-        raise Exception("sensors_init failed")
+    result = _hdl.sensors_init(file)
+    if result != 0:
+        raise_sensor_error(reslt, "sensors_init failed")
```

Comment:
reslt

Reply:
Doh!  Apparently not checked well enough.